src/main/java/uk/gov/ida/eidas/trustanchor/FileKeyLoader.java

```diff
@@ -24,8 +27,18 @@ public static RSAPrivateKey load(final File keyFile) throws InvalidKeySpecExcept
     return (RSAPrivateKey) keyFactory.generatePrivate(new PKCS8EncodedKeySpec(Files.readAllBytes(keyFile.toPath())));
   }
 
-  public static X509Certificate loadCert(final File certFile) throws CertificateException, FileNotFoundException {
-    final CertificateFactory certificateFactory = CertificateFactory.getInstance(CERTIFICATE_TYPE);
-    return (X509Certificate) certificateFactory.generateCertificate(new FileInputStream(certFile));
+  public static X509Certificate loadCert(final File certFile) {
+    try {
+      CertificateFactory certificateFactory = CertificateFactory.getInstance(CERTIFICATE_TYPE);
+      return (X509Certificate) certificateFactory.generateCertificate(new FileInputStream(certFile));
+    } catch (Exception e) {
+      throw new RuntimeException(e);
```

Comment:
Is this standard practice in Java? I know it's necessary to use streams, which is why I've resisted using them.

Reply:
Better to throw a new Exception that wraps RuntimeException e.g. `new  KeyLoadingException(e)`